components/Form/index.js

```diff
@@ -59,21 +63,28 @@ export default function Form({ onSubmit, isEditing, eventToEdit }) {
           type="text"
           id="event"
           name="event"
+          placeholder="e.g. Wedding, Conference"
           defaultValue={eventToEdit?.event}
           required
+          aria-label="Enter an event"
         />
         <label htmlFor="date">Date:</label>
         <input
           type="datetime-local"
           id="date"
           name="date"
-          defaultValue={eventToEdit?.date}
+          defaultValue={
+            eventToEdit?.date
+              ? new Date(eventToEdit.date).toISOString().slice(0, 16)
```

Comment:
I would create a `formatDateForInput` function and pass `eventToEdit.date ` to it - the less JS in JSX, the better!